lib/keila_web/views/api_view.ex

```diff
@@ -0,0 +1,125 @@
+defmodule KeilaWeb.ApiView do
+  use KeilaWeb, :view
+  alias Keila.Pagination
+  import KeilaWeb.ApiNormalizer.SchemaMapper
+
+  def render("contacts.json", %{contacts: contacts = %Pagination{}}) do
+    %{
+      "meta" => %{
+        "page" => contacts.page,
+        "pageCount" => contacts.page_count,
+        "count" => contacts.count
+      },
+      "data" => Enum.map(contacts.data, &contact_data/1)
+    }
+  end
+
+  def render("contact.json", %{contact: contact}) do
+    %{
+      "data" => contact_data(contact)
+    }
+  end
+
+  def render("campaigns.json", %{campaigns: campaigns = %Pagination{}}) do
+    %{
+      "meta" => %{
+        "page" => campaigns.page,
+        "pageCount" => campaigns.page_count,
+        "count" => campaigns.count
+      },
+      "data" => Enum.map(campaigns.data, &campaign_data/1)
+    }
+  end
+
+  def render("campaign.json", %{campaign: campaign}) do
+    %{"data" => campaign_data(campaign)}
+  end
+
+  def render("segments.json", %{segments: segments = %Pagination{}}) do
+    %{
+      "meta" => %{
+        "page" => segments.page,
+        "pageCount" => segments.page_count,
+        "count" => segments.count
+      },
+      "data" => Enum.map(segments.data, &segment_data/1)
+    }
+  end
+
+  def render("segment.json", %{segment: segment}) do
+    %{"data" => segment_data(segment)}
+  end
+
+  def render("errors.json", %{errors: errors}) do
+    %{
+      "errors" => Enum.map(errors, &error_object/1)
+    }
+  end
+
+  defp contact_data(contact) do
```

Comment:
What do you think in separating this _data methods on the correspondent modules?

In this way we can tie together the map JSON to the contact schema.